bower.json

```diff
@@ -1,8 +1 @@
-{
-  "name": "bespin-ui",
-  "dependencies": {
-    "ember": "~2.9.0",
-    "ember-cli-shims": "0.1.3",
-    "bootstrap": "~3.3.5"
-  }
-}
+{"name":"bespin-ui","dependencies":{}}
```

Comment:
Do we need this file?
It looks like the dependencies are now in [package.json](https://github.com/Duke-GCB/bespin-ui/pull/32/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2) under `devDependencies`.